railties/lib/rails/engine/lazy_route_set.rb

```diff
@@ -87,19 +87,19 @@ def routes
         def method_missing_module
           @method_missing_module ||= Module.new do
             private
-              def method_missing(method_name, *args, &block)
+              def method_missing(...)
```

Comment:
I don’t know if this change is necessary since the `super` call below no longer explicitly passes `*args`, however it seemed more idiomatic since the arguments are simply delegated to `public_send` and this is supported in the now minimum version, `3.1`.